mesonbuild/interpreter.py

```diff
@@ -2824,7 +2824,10 @@ def add_languages_for(self, args, required, for_machine: MachineChoice):
                         raise
             if machine_name != 'build' or self.environment.is_cross_build():
                 mlog.log(comp.get_display_language(), 'compiler for the', machine_name, 'machine:',
-                         mlog.bold(' '.join(comp.get_exelist())), comp.get_version_string())
+                        mlog.bold(' '.join(comp.get_exelist())), comp.get_version_string())
```

Comment:
__[Flake8]__ [E128] continuation line under-indented for visual indent ([link](https://sider.review/gh/repos/19784232/pulls/5681#issue-4423135))

<sub>posted by Sider</sub>